### DISCUSSION TITLE
Comments not showing, but unsure if this is a config issue or something else

### DISCUSSION THREAD

**Discussion by piperhaywood** [Automattic/wordpress-activitypub#543] [Q&A] [answered]
Comments that folks leave on my posts on Mastodon don’t seem to be coming through on my WP site. I had a rocky start to getting ActivityPub and Webfinger set up on my site due to my hosting provider’s NGINX setup ([more on this here](https://github.com/Automattic/wordpress-activitypub/discussions/446)), so I’m wondering, is there possibly something else I need to tweak in the NGINX config to get comments working?

**Comment by pfefferle** [comment, marked as answer]
Normally not, have you checked the pending, spam and trash folders?

**Reply by piperhaywood** [reply]
Ohhhh interesting, it went straight to trash... I had a look at my “Disallow comment keys” and it is super long, something in there is causing the problem I’m sure. Will sort it out there, sorry for the bother! Thanks!
